source/about/embed-mattermost-within-microsoft-teams.rst

```diff
@@ -1,20 +1,15 @@
-Install the Mattermost App in Microsoft Teams
-=============================================
-
+Embed Mattermost within Microsoft Teams
+=======================================
 .. include:: ../_static/badges/allplans-cloud-selfhosted.rst
   :start-after: :nosearch:
 
-.. tip::
-
-  Looking to `install the Mattermost for Microsoft Teams plugin </about/install-mattermost-for-microsoft-teams-plugin.html>`__ instead?
-
-With `Mattermost for Microsoft Teams </about/mattermost-for-microsoft-teams.html>`__, you can embed your Mattermost workspace within your Microsoft Teams instance and take advantage of `embedded app features <#benefits-of-the-embedded-app>`_.
+With the `Mattermost for Microsoft Teams </about/mattermost-for-microsoft-teams.html>`__ integration, you can embed your Mattermost workspace within your Microsoft Teams instance and take advantage of `embedded app features <about/mattermost-for-microsoft-teams.html#benefits-of-embedding-mattermost>`__.
```

Comment:
"Embedded Mattermost for Microsoft Teams"

Reply:
Should we capitalize "embedded"?

Reply:
This page is specific to the embedded experience, and this paragraph is introducing the connection between the integration itself and the embedded feature.